RabbitMQ.Stream.Client/RawSuperStreamProducer.cs

```diff
@@ -14,22 +14,22 @@
 namespace RabbitMQ.Stream.Client;
 
 /// <summary>
-/// SuperStreamProducer is a producer that can send messages to multiple streams.
+/// RawSuperStreamProducer is a producer that can send messages to multiple streams.
 /// Super Stream is available in RabbitMQ 3.11.0 and later.
 /// SuperStreamProducer is actually an abstraction over multiple Producers.
 /// It does not contain any connection or state, it is just a way to send messages 
 /// to multiple streams using multi producers.
 /// When a message is sent to a stream, the producer will be selected based on the stream name and the partition key.
 /// SuperStreamProducer uses lazy initialization for the producers, when it starts there are no producers until the first message is sent.
```

Comment:
Perhaps we can add a link to the docs here in this doc-comment?
https://rabbitmq.com/streams.html#super-streams